lib/node_modules/@stdlib/complex/parse-float64/docs/repl.txt

```diff
@@ -0,0 +1,27 @@
+
+{{alias}}( str )
+    Parses a string representation of a 128-bit complex number.
+
+    Parameters
+    ----------
+    str: string
+        String representation of a complex number.
+
+    Returns
+    -------
+    out: Complex128
+        If the string is recognized as a complex number, a Complex128 instance is returned.
+
+    Throws
+    ------
+    Error
+        If the string is not recognized as a complex number, an error is thrown.
+
```

Comment:
```suggestion
```

See our repl text guide: https://github.com/stdlib-js/stdlib/blob/develop/docs/repl_text.md. We don't include a throws section.